src/node/mod.rs

```diff
@@ -319,6 +319,14 @@ impl Node {
             .flat_map(|stage| stage.shared_state.our_adults())
     }
 
+    /// Returns the adults of our section sorted by their distance to `name` (closest first).
+    /// If we are not elder or if there are no adults in the section, returns empty vec.
```

Comment:
the comment says `If we are not elder ... returns empty vec `
so just want to confirm that adult won't have the info of other adults within the same section?

Reply:
Yes, only elders know about non-elders.